R/plot.R

```diff
@@ -982,6 +982,10 @@ plot_MSEv_eval_crit <- function(explanation_list,
     stop("ggplot2 is not installed. Please run install.packages('ggplot2')")
   }
 
+  if (!requireNamespace("data.table", quietly = TRUE)) {
+    stop("data.table is not installed. Please run install.packages('data.table')")
+  }
+
```

Comment:
I don't think this is necessary as data.table is already in imports of the shapley package, so not possible to install the shapr package without data.table.

Reply:
Removed it here and in the MSEv plot function, too.